### PR TITLE
TASK: Remove google plus support

### PR DESCRIPTION
**Pull request by markusguenther**
Google plus is offline for non company related things and not supported anymore.

Bildschirmfoto 2019-11-02 um 17 17 51

### CODE REVIEW

**Comment by Sebobo** [comment]
Looks good by reading

File: DistributionPackages/Neos.NeosIo/Classes/Command/CrowdCommandController.php [hunk 1]
```diff
@@ -164,7 +164,7 @@ public function setUserAttributeCommand($userName, $attribute, $value)
      *
      * Structure has to similiar to this:
      *
-     * username;neos_googleplus;neos_email;neos_twitter;...
+     * username;neos_email;neos_twitter;...
      * xyz;xyz;xyz@example.org;@xyz;...
```
Comment (kdambekalns) [inline DistributionPackages/Neos.NeosIo/Classes/Command/CrowdCommandController.php]:
```suggestion
     * xyz;xyz@example.org;@xyz;...
```

Comment (Sebobo) [reply DistributionPackages/Neos.NeosIo/Classes/Command/CrowdCommandController.php]: I don't think it matters. Actually we could just remove the command as by now it's easier to change the values in crowd directly which was not possible at the time.

Comment (markusguenther) [reply DistributionPackages/Neos.NeosIo/Classes/Command/CrowdCommandController.php]: I could also split it up. First intention was to remove the button on the page and then I found more and more google plus stuff ;)

**Pull request merged**